index.html

```diff
@@ -3458,6 +3458,36 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ________ Contributor's card END ________  -->
 
+        <!-- ________ Contributor's card START ________  -->
+        <div class="card">
+
+            <p class="name">Aloys B</p>
+            <p class="contact">
+                <i class="fa fa-twitter"></i>
+                <a href="YOUR twitter account link" target="_blank">None :smile:</a>
+            </p>
+            <p class="about">
+                Another student of web development, dreaming of remote work ... Paraglider, sailor, kitesurfer, student of life and student FOR life.
+            </p>
+            <div class="resources">
+                <p>3 Useful Dev Resources</p>
+                <ul>
+                    <li>
+                        <a href="#" target="_blank" title="www.udemy.com">Udemy</a>
```

Comment:
The url to the website needs to be in the `href` and not the `title`.
So it would look like this 
```suggestion
                        <a href="www.udemy.com" target="_blank" title="online courses">Udemy</a>
```
YOu need to do the same for the other links too.